vars/provisionNodes.groovy

```diff
@@ -256,8 +270,12 @@ EOF'''
       if (inst_rpms) {
          provision_script += '''\nyum -y erase ''' + inst_rpms
       }
-      provision_script += '\nrpm --import https://download.fedoraproject.org/' +
-                                         'pub/epel//RPM-GPG-KEY-EPEL-7'
+      gpg_key_urls.each { gpg_url ->
+        provision_script += '\nrpm --import ' + gpg_url
+      }
+      provision_script += '\nrpm --import ' +
+              'https://copr-be.cloud.fedoraproject.org/results/jhli' +
+              '/ipmctl/pubkey.gpg'
```

Comment:
Given that this is project (i.e. DAOS) specific, it would be good to figure out a way to push this out to that project, at some point in the future and keep this routine as generic as possible.  Realising of course we are already down the road of non-genericness.  But probably much of that can be eliminated with the `inst_repos` and `inst_rpms` support added subsqeuently (to the non-genericness being added) or something like it for full repo paths and keys.